javaslang/src-gen/test/java/javaslang/CheckedFunction2Test.java

```diff
@@ -165,15 +165,6 @@ public void shouldLiftTryPartialFunction() {
         assertThat(unknown.getCause().getMessage()).isEqualToIgnoringCase("Unknown MessageDigest not available");
     }
 
-    private static final CheckedFunction2<Integer, Integer, Integer> recurrent1 = (i1, i2) -> i1 <= 0 ? i1 : CheckedFunction2Test.recurrent2.apply(i1 - 1, i2) + 1;
```

Comment:
Force pushed a new version, all tests seem to be passing now on `Java 9` also (from Idea only, not yet from Maven).
Thanks for your help @ruslansennov, @zsolt-donca.
@danieldietrich, do you think we should add more tests here?